site/content/docs/latest/tutorials/kubeapps-on-tce/README.md

```diff
@@ -2,6 +2,6 @@
 
 [VMware Tanzu™ Community Edition (TCE)](https://tanzucommunityedition.io) is a full-featured, easy-to-manage Kubernetes platform for learners and users, especially those working in small-scale or preproduction environments.
 
-[Kubeapps](https://kubeapps.com/) provides a web-based dashboard to deploy, manage, and upgrade applications on a Kubernetes cluster. It is a one-time install that gives you a number of important benefits, including the ability to browse and deploy Helm charts from public or private chart repositories and upgrade, manage and delete deployments from your Web browser.
+[Kubeapps](https://kubeapps.com/) provides a web-based dashboard to deploy, manage, and upgrade applications on a Kubernetes cluster. It is a one-time install that gives you a number of important benefits, including the ability to browse and deploy Helm charts and Carvel packages from public or private repositories and upgrade, manage and delete deployments from your Web browser.
```

Comment:
If we want to add Carvel here (or sth cooler like "any package"), we might want to replicate this change to: 
- Repo description
- Chart metadata
- Chart readme + bitnami
- Website

Reply:
Agree.